src/core/Form/Combobox/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,595 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv } from '../../../../reset';
+import { windowAvailable } from '../../../../utils/common';
+import { AutoId } from '../../../../utils/AutoId';
+import { Debounce } from '../../../utils/Debounce/Debounce';
+import { Button } from '../../../Button/Button';
+import { Chip } from '../../../Chip';
+import { Popover } from '../../../Popover/Popover';
+import { FilterInput, FilterInputStatus } from '../../FilterInput/FilterInput';
+import { ComboboxItemList } from '../ComboboxItemList/ComboboxItemList';
+import { ComboboxItem } from '../ComboboxItem/ComboboxItem';
+import { ComboboxEmptyItem } from '../ComboboxEmptyItem/ComboboxEmptyItem';
+import { ChipList } from '../ChipList/ChipList';
+import { baseStyles } from './Combobox.baseStyles';
+
+const baseClassName = 'fi-combobox';
+const comboboxClassNames = {
+  wrapper: `${baseClassName}_wrapper`,
+  open: `${baseClassName}--open`,
+  removeAllButton: `${baseClassName}_removeAllButton`,
+  error: `${baseClassName}--error`,
+};
+
+export interface ComboboxData {
+  /** Unique label that will be shown on combobox item and used on filter */
+  labelText: string;
```

Comment:
Does this need to be text or could it be ReactNode?

Reply:
Then the filter functionality and highlighting feature needs changes to support that. And also the `chipText` etc.

At this point I don't see a reason to introduce more complex logic and increasing changes for user to break accessibility of the component.

But of course if this is needed / seen to prove enough value with low effort, then we could change it. Now or later in separate PR's.

Reply:
Makes sense, I didn't take that into account. Lets leave it as it is!